build-tools/installers/create-installers.targets

```diff
@@ -432,7 +432,6 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Installer.Common.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Installer.Common.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Installer.Common.targets" />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\K4os.Compression.LZ4.dll" />
```

Comment:
Were the two assemblies here listed twice?

Reply:
Yeah, I had an initial take where I was marking third party items in this file and found a couple duplicates that should be cleaned up as part of that.